app/controllers/users_controller.rb

```diff
@@ -11,9 +11,9 @@ class UsersController < ApplicationController
   def admin_index
     authorize User
     if current_user.can_super_admin?
-      @users = User.includes(:roles).page(1)
+      @users = User.includes(:perms).page(1)
```

Comment:
the User model has default_scope including :org and :perms already so is not really needed to include :perms there